index.html

```diff
@@ -1,287 +1,292 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<meta charset="UTF-8">
-<meta http-equiv="X-UA-Compatible" content="IE=edge">
-<meta name="viewport" content="width=device-width, initial-scale=1.0">
-<title>Portfolio Trial</title>
-<link rel="stylesheet" href="style.css"/>
-<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css"/>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet"/>
+  <title>Portfolio (mobile and desktop version)</title>
 </head>
 <body>
-<section class = "toolbar-section">
-<div class="font">
-    <div class="container-1">
-        <div class="font">
-            <header class="header">
-                <a class="myLogo"> My Logo </a>
-                <nav class="burger">
-                    <ul class="ul-image">
-                        <li>
-                            <a href="#">
-                <img src="/Images/Union.png" alt="union-background"/>
-            </a>
+  <div class="firstpage">
+    <header class="navbar">
+      <a class="myLogo"> My Logo </a>
+      <div class="burger">
+        <ul class="ul-image">
+            <li>
+                <a href="#">
+                    <img src="./Images/Union.png" alt="union-background" />
+                </a>
             </li>
         </ul>
-        </nav>
+      </div>
+      <nav class="ul-menu">
+        <ul class="items-nav">
+          <li class="vers-desktop"><a href="#Portfolio">Portfolio</a></li>
+          <li class="vers-desktop"><a href="#About">About</a></li>
+          <li class="vers-desktop"><a href="Contact">Contact</a></li>
+        </ul>
+      </nav>
     </header>
 
-    <section class="headline-section">
-        <div>
-            <h2 class="first-title">I'm Peter <br/> Glad to see you! </h2>
-        </div>
-        <div>
-            <p class="description">I’m a software developer! I can help you build a product ,
-                feature or website Look through some of my work and experience!
-                If you like what you see and have a project you need coded,
-                don’t hestiate to contact me.
+    <section>
+      <div class="container-1">
+        <section>
+          <h2 class="first-title">I’m Peter <br/>Glad to see you!</h2>
+            <p class="description">
+              I’m a software developer! I can help you build a product , 
+              feature or website Look through some of my work and experience! 
+              If you like what you see and have a project you need coded, 
+              don’t hestiate to contact me.
             </p>
-        </div>
-        <p class="description2">LET'S CONNECT</p>
+        </section>
+        <div class="section2"> 
+          <h3 class="connect">LET’S CONNECT</h3>
```

Comment:
- [ ] on Tablet devices you `<h3>` tag (let's connect) get overflow over the text please try to fix that 
![image](https://user-images.githubusercontent.com/61790587/177787984-3ef2932f-a588-45da-b287-15ede11818b7.png)